examples/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -2201,7 +2380,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/examplepbBody"
+              "type": "object"
```

Comment:
the only field in the request object is captured by a path parameter.

Reply:
Is there some way we could remove this `"in": "body"` entry altogether?